plugins/postgres/dbt/adapters/postgres/relation.py

```diff
@@ -1,4 +1,26 @@
 from dbt.adapters.base import Column
+from dataclasses import dataclass
+from dbt.adapters.base.relation import BaseRelation
+from dbt.exceptions import RuntimeException
+
+
+@dataclass(frozen=True, eq=False, repr=False)
+class PostgresRelation(BaseRelation):
+    def __post_init__(self):
+        # Check for length of Postgres table/view names.
+        # Check self.type to exclude test relation identifiers
+        if (
+            self.identifier is not None
+            and self.type is not None
+            and len(self.identifier) > self.relation_max_name_length()
+        ):
+            raise RuntimeException(
+               f"Postgres relation name '{self.identifier}' is longer than "
```

Comment:
Tiny thing: Including `Postgres` as a literal here means that we see the following error on Redshift:
```
  Postgres relation name 'tell_me_about_a_complicated_man_muse_tell_me_how_he_wandered_and_was_lost_when_he_had_wrecked_the_holy_town_of_troy_and_where_he_went_and_who_he_met_the_pain_he_suffered_in_the_storms_at_sea' is longer than 127 characters
```

This is not a common error on Redshift (that's not a model name, that's a tweet!). And Redshift users are used to seeing postgres errors, though it's confusing in this case because Redshift + Postgres have different limits.

I think it'd be better to make this either:
* `f"{adapter.name} relation name '{self.identifier}' ...` (not sure if possible here)
* `f"Relation name '{self.identifier}' ...` (exclude entirely)

Reply:
I've removed the 'Postgres' and fixed the formatting flake8 was complaining about.